### ISSUE TITLE
Don't ask for full files access 

### ISSUE THREAD

**Issue by aibosss** [j-fbriere/squawker#29]
### Describe the problem

Don't need that to allow auto download. 

### Describe the solution

Don't need that to allow auto download.

**Comment by j-fbriere** [comment]
I'm not sure I understand.
What kind of file access should be granted?

**Comment by aibosss** [comment]
Look at this screenshot : postimg . cc/RJJWCsHw

Everytime we want to download 1 pic or 1 vid, it asks for full phone files access. But then if we deny, it actually saves the file anyway. So, just stop asking for full files access all along .

**Comment by ahuj9** [comment]
I agree, full files special permission is completely inappropriate for this app

**Comment by aibosss** [comment]
Here is how QuaX fixed this, surely you can please fix this 
https://github.com/Teskann/QuaX/commit/5215f9e5ca5ae5aa587b5df5ec93d345728c687b

**Comment by j-fbriere** [comment]
I'll have a look at it. Thank you.

**Comment by Runmek** [comment]
Offering 10$ donation for this fix

**Comment by j-fbriere** [comment]
3.8.7 should fix it.
